src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -54,7 +54,8 @@ public void periodic()
   }
 
   @Override
-  public void simulationPeriodic() {
+  public void simulationPeriodic()
+  {
     // This method will be called once per scheduler run during simulation
   }
```

Comment:
delete if not needed.  The `ExampleSubsystem` has them if anyone wants to see the template